runtimevar/runtimevar.go

```diff
@@ -248,16 +257,17 @@ func (c *Variable) background(ctx context.Context) {
 //
 // Latest returns ErrClosed if the Variable has been closed.
 func (c *Variable) Latest(ctx context.Context) (Snapshot, error) {
-	var haveGood bool
+	var haveLatest bool
 	select {
 	case <-c.haveGood:
-		haveGood = true
+		haveLatest = true
 	case <-ctx.Done():
+		haveLatest = c.haveLatest()
```

Comment:
I see what you're doing here, but I think it might be more clear like this:

```
haveGood := c.haveGood()
if !haveGood {
  select {
    case <-c.haveGood:
      haveGood = true
    case <-ctx.Done():
      // We don't return ctx.Err().
  }
  ...
```

WDYT?

Reply:
Sure, note that we can't have `c.haveGood()` due to `<-c.haveGood`. You ok with `.haveLatest()`?